client/src/pages/dashboards/DecisivesDashboard.js

```diff
@@ -0,0 +1,338 @@
+      })
+      .addVariable("locationQuery", "LocationSearchQueryInput", {
+        pageNum: 0,
+        pageSize: 0,
+        status: "ACTIVE"
+      })
+      .addVariable("taskQuery", "TaskSearchQueryInput", {
+        pageNum: 0,
+        pageSize: 0,
+        status: "ACTIVE"
+      })
+
+    const dashboardSettings = Settings.dashboards.find(
+      o => o.label === this.props.match.params.dashboard
+    )
+
+    fetch(dashboardSettings.data)
+      .then(response => response.json())
+      .then(dashboardData =>
+        GQL.run([dataPart]).then(data => {
+          this.setState({
+            decisives:
+              dashboardData &&
+              dashboardData.map(decisive => {
+                return {
+                  label: decisive.label,
+                  positions: data.positionList.list.filter(item =>
+                    decisive.positions.includes(item.uuid)
+                  ),
+                  tasks: data.taskList.list.filter(item =>
+                    decisive.tasks.includes(item.uuid)
+                  ),
+                  locations: data.locationList.list.filter(item =>
+                    decisive.locations.includes(item.uuid)
+                  )
+                }
+              }),
+            isLoading: !this.state.reports
+          })
+        })
+      )
+  }
+
+  render() {
+    if (!this.state.decisives || !this.state.reports) return null
+    return (
+      <>
+        <Panel>
+          <Panel.Heading>
+            <Panel.Title componentClass="h3">People</Panel.Title>
+          </Panel.Heading>
+          <Panel.Body>
+            {this.state.decisives.map(decisive => (
+              <StatsTable
+                label={decisive.label}
+                key={decisive.label}
+                data={decisive.positions}
+                contentData={this.state.positionStats}
+                prevContentData={this.state.prevPositionStats}
+                itemLabel={item => <LinkTo position={item}>{item.name}</LinkTo>}
+              />
+            ))}
+          </Panel.Body>
+        </Panel>
+        <Panel>
+          <Panel.Heading>
+            <Panel.Title componentClass="h3">Places</Panel.Title>
+          </Panel.Heading>
+          <Panel.Body>
+            {this.state.decisives.map(decisive => (
+              <StatsTable
+                label={decisive.label}
+                key={decisive.label}
+                data={decisive.locations}
+                contentData={this.state.locationStats}
+                prevContentData={this.state.prevLocationStats}
+                itemLabel={item => (
+                  <LinkTo anetLocation={item}>{item.name}</LinkTo>
+                )}
+              />
+            ))}
+          </Panel.Body>
+        </Panel>
+        <Panel>
+          <Panel.Heading>
+            <Panel.Title componentClass="h3">Processes</Panel.Title>
+          </Panel.Heading>
+          <Panel.Body>{/* <StatsTable data={[]} /> */}</Panel.Body>
```

Comment:
This part of the dashboard would appear to be always empty.

Reply:
yeah, waiting for more requirements on that one... still work in progress